app/views/events/upcoming.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1>Upcoming Events</h1>
+
+<% @events.each do |event| %>
+  <div>
+    <h2><%= link_to event.name, event_path(event) %></h2>
+
+    <dl>
```

Comment:
Not the best use case of a `<dl>` here. I'd rather see this be an unordered list.